parts/k8s/kubernetesmastervars.t

```diff
@@ -236,6 +256,18 @@
     "masterFirstAddrOctets": "[split(parameters('firstConsecutiveStaticIP'),'.')]",
     "masterFirstAddrOctet4": "[variables('masterFirstAddrOctets')[3]]",
     "masterFirstAddrPrefix": "[concat(variables('masterFirstAddrOctets')[0],'.',variables('masterFirstAddrOctets')[1],'.',variables('masterFirstAddrOctets')[2],'.')]",
+    "masterEtcdServerPort": {{GetMasterEtcdServerPort}},
+    "masterEtcdClientPort": {{GetMasterEtcdClientPort}},
+    {{if IsMasterVirtualMachineScaleSets}}
+    "masterVMNamePrefix": "[concat(parameters('orchestratorName'), '-master-', parameters('nameSuffix'), '-')]",
+    "masterVMNames": [
```

Comment:
I don't think `masterVMNames` is ever used by VMSS masters.

Reply:
This was added in Kubernetesmastersvmss. So I had it copied it over from that file. Do you mean that it is okay to remove?

Reply:
Yes, I meant it's not being used currently. So we should remove it.